src/02-common-principles.md

```diff
@@ -161,6 +161,27 @@ A summary of all entities in BIDS and the order in which they MUST be
 specified is available in the [entity table](./99-appendices/04-entity-table.md)
 in the appendix.
 
+### Entity-linked file collections
+
+A set of files that are related to each other based on a repetitive acquisition of
+sequential data by changing acquisition parameters one (or multiple) at a time and/or by
+being inherent components of the same data.
+The suffix serves as an identifier for these
+collections, holding a logical grouping of constituent files.
```

Comment:
I don't quite understand this sentence. I think "holding" means that the suffix indicates which set of files to expect. What about:

```suggestion
Entity-linked collections are identified by a common suffix,
indicating the group of files that should be considered a logical unit.
```